scripts/errcheck_excludes.txt

```diff
@@ -0,0 +1,7 @@
+(github.com/prometheus/alertmanager/vendor/github.com/go-kit/kit/log.Logger).Log
+// Allowed log levels are enforced via kingpin
+(*github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/promlog.AllowedLevel).Set
+
+(io.Closer).Close
+(*os.File).Close
+(io.Writer).Write
```

Comment:
Does this catch the response writer above that you added the error logging to?

Reply:
Removed, wasn't needed.